### ISSUE TITLE
Feature request: Debian Packaging

### ISSUE THREAD

**Issue by cyberjacob** [cavaliercoder/zabbix-module-systemd#22]
Packaging as a .deb file would be useful :)

**Comment by zentavr** [comment]
I have a working code which creates a .deb

**Comment by GleitonCampos** [comment]
I agree with the .deb file. I know that this is a "lazy" way, but will be very useful to install in many servers/clients

**Comment by cavaliercoder** [comment]
@zentavr Could you please raise a PR with your deb code?

**Comment by cyberjacob** [comment]
@GleitonCampos Considering Zabbix is designed to monitor hundreds of servers, I'd say this is more "using the package manager as intended" rather than lazy...

**Comment by zentavr** [comment]
@cyberjacob: I have composed deb packaging for another your module called **zabbix-agent-module-sockets**. As a dependency it contains zabbix sources inside (as an tar.gz file).

I'm ok to share it , but I wonder if I can to that with zabbix sources inside? If not - I can just drop them and share the files.

Frankly speaking just now I have no time to make it more elegant, but if you have time to add the files in more proper way - it would be great.

**Comment by cavaliercoder** [comment]
I apologize I haven't had the bandwidth to do this. Package maintaining tends to require a lot of commitment that I can't offer right now (testing on multiple-OSes is the hardest part) - but I acknowledge and am grateful that you all make use of this library.

@zentavr You can share your package code, without sharing the tar file. Ideally, a Makefile should be used to compile the tar and the sources should be provided by the user. I have an examples in another modules:
* https://github.com/cavaliercoder/libzbxpgsql-build/tree/master/debuild
* https://github.com/cavaliercoder/libzbxpython-build/tree/master/debian

**Comment by zentavr** [comment]
@cavaliercoder - the changes are just **debian** folder which contains all the rules. After that I build it with the next:
```
dpkg-buildpackage -rfakeroot -us -uc
```
